sriovnet_switchdev.go

```diff
@@ -65,6 +65,13 @@ func GetUplinkRepresentor(vfPciAddress string) (string, error) {
 	}
 	for _, device := range devices {
 		if isSwitchdev(device.Name()) {
+			// Try to get the phys port name, if not exists then fallback to check without it
```

Comment:
can you explain the the comment that uplink representor phys port name is expected to be in the format of `p<port-num>` e.g `p0`